### PR TITLE
Translate SHOW/HIDE password button

### PR DESCRIPTION
**Pull request by BryanEliDimas**
Add translation for the SHOW/HIDE password button

### CODE REVIEW

File: app/views/newflow/login_signup/_password_show_hide.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <span id="password-show-hide-button">
-    <span class="toggle-show-hide">SHOW</span>
-    <span class="toggle-show-hide" style="display: none">HIDE</span>
+    <span class="toggle-show-hide">
+      <%= I18n.t(:"login_signup_form.show_password_btn")  %>
+    </span>
+    <span class="toggle-show-hide" style="display: none">
```
Comment (steveburkett) [inline app/views/newflow/login_signup/_password_show_hide.html.erb:5]: do we want style the erb?  this seems kinda controversial.  just asking.

Comment (BryanEliDimas) [reply app/views/newflow/login_signup/_password_show_hide.html.erb:5]: If I understand your question correctly, yes, that is intentional. I don't know how else to make that work.

**Comment by BryanEliDimas** [comment]
@steveburkett  Please re-review?

File: app/views/newflow/login_signup/_password_show_hide.html.erb (hunk 1 above)
Comment (steveburkett) [reply app/views/newflow/login_signup/_password_show_hide.html.erb:5]: unobtrusive css :)   to start with a class that denotes hidden.

**Pull request merged**